### PR TITLE
Update storybook for multi accordion change

### PR DESCRIPTION
**Pull request by bkjohnson**
## Description
Related to #124 

Make sure the feature is appropriately reflected in Storybook

**Note**: Before this is merged, I will need to merge 124 and make a release so that I can use the tagged release as opposed to the feature branch name

## Testing done

Storybook :eyes: 

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "build": "node ./scripts/build.js",
     "build-storybook": "build-storybook",
     "prepublishOnly": "pinst --disable && node ./scripts/prepublish.js",
-    "postinstall": "husky install && cd node_modules/web-components && yarn install --production && npm run-script build",
+    "postinstall": "husky install",
```
Comment (bkjohnson) [inline package.json:22]: With the `prepare` script that was part  of the most recent release, we no longer need to install + build in the `postinstall`

File: package.json [hunk 2]
```diff
@@ -97,7 +97,7 @@
     "sass-loader": "^8.0.2",
     "sinon": "^9.2.2",
     "uswds": "1.6.10",
-    "web-components": "https://github.com/department-of-veterans-affairs/component-library.git#wc-v0.7.0"
+    "web-components": "https://github.com/department-of-veterans-affairs/component-library.git#wc-accordion-multi-patch"
```
Comment (bkjohnson) [inline package.json:100]: Need to change this to the tagged release before merge.

File: .github/workflows/publish-storybook.yml [hunk 3]
```diff
@@ -1,15 +1,15 @@
 name: Build storybook & deploy
-on:
-  push:
-    branches:
-      - master
+on: push
+  # push:
+  #   branches:
+  #     - master
```
Comment (cvalarida) [inline .github/workflows/publish-storybook.yml:5]: We've gotta find a better way to test these things. When making changes to the GH action workflow, this makes enough sense, but it doesn't look like that's what you're doing here.

File: stories/va-accordion.stories.jsx [hunk 4]
```diff
@@ -77,10 +76,10 @@ Default.args = {
 };
 Default.argTypes = propStructure(accordionDocs);
 
-export const Multi = Template.bind({});
-Multi.args = {
+export const NoMulti = Template.bind({});
+NoMulti.args = {
```
Comment (cvalarida) [inline stories/va-accordion.stories.jsx:80]:
```suggestion
export const OnlyOpenOne = Template.bind({});
OnlyOpenOne.args = {
```
I'm thinking of the sidebar nav menu. Seeing "No Multi" is maybe a bit awkward. "Open Only One" is maybe less awkward? I'm game to change it, but our goal is to make it obvious to a new engineer looking at Storybook for the first time to know what all the things do by the story name.

**Review by cvalarida — COMMENTED** [review]
I'll wait to approve until you change back the `publish-storybook.yml` and add the right `web-components` release, but this looks good otherwise.

File: .github/workflows/publish-storybook.yml (hunk 3 above)
Comment (bkjohnson) [reply .github/workflows/publish-storybook.yml:5]: I agree. I'm going to look into chromatic again.

**Review by cvalarida — APPROVED** [review]

**Review by cvalarida — APPROVED** [review]

**Pull request merged**